docs/converter/shared_library.md

```diff
@@ -5,7 +5,7 @@ After you generate your project from your protocol buffer, you can build librari
 The compile command would be like below.
 
 ```
-make lib_x86
+make libdlk_x86
```

Comment:
target name of make is not changed.

Reply:
oh. Thanks for noticing that! I'll revert this change.